performancetest/src/main/java/org/corfudb/performancetest/StreamPerformanceTest.java

```diff
@@ -0,0 +1,108 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.stream.IStreamView;
+import org.junit.Test;
+import java.util.Properties;
+
+/**
+ * Created by Nan Zhang and Lin Dong on 10/23/19.
+ */
+
+public class StreamPerformanceTest {
+    private String endPoint = "localhost:9000";
+    private int metricsPort = 1000;
+    private static final Properties PROPERTIES = new Properties();
```

Comment:
load properties, now you are using default values.

Reply:
Same you can change it directly.